app/models/profile.rb

```diff
@@ -92,6 +92,13 @@ def activate_after_passing_in_person
     activate
   end
 
+  def activate_after_password_reset
+    update!(
+      deactivation_reason: nil,
```

Comment:
Should we check here that`deactivation_reason` is `password_reset` before removing it?